dist/field/field.css

```diff
@@ -63,8 +63,14 @@ div.field__group {
 .field__control + span.field__description {
   margin-left: 8px;
 }
+.field__description .icon {
+  display: inline-block;
+  margin-right: var(--spacing-50);
+}
 div.field__description {
-  margin: 2px 0 0;
+  margin-left: 0;
+  margin-right: 0;
+  margin-top: var(--spacing-100);
```

Comment:
This can still be 1 line. 
Why not do `margin: var(--spacing-100) 0 0`, like it was before. Do we not want bottom margin

Reply:
Actually, bottom wasn't resetting any margins, but I'll go ahead and change it. I guess it can be a safeguard for resetting.